src/memex/search/config.py

```diff
@@ -241,6 +237,19 @@ def update_aliased_index(client, new_target):
     })
 
 
+def update_index_settings(client):
```

Comment:
Yeah, sorry, I should have added a comment about this. I think we should add a CLI command to run this manually. Doing it automatically doesn't make a lot of sense to me, especially if it's only going to happen in dev -- seems like it'll just end up as a way to catch people out.